### PR TITLE
Checklist: add pbs-checklist (basis, aufbau, wolf, pfadi)

### PR DESCRIPTION
**Pull request by pmattmann**
PBS-Checklist für
- Basis Wolf
- Basis Pfadi
- Aufbau Wolf
- Aufbau Pfadi

### CODE REVIEW

File: api/migrations/schema/checklists/pbs-checklist.sql [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000300040005', '000300000000', '000300040000', 5, 'Betreuung und Förderung einzelner Teammitglieder')
+;
+
+DELETE FROM public.checklist c where c.id = '000400000000';
+INSERT INTO public.checklist (createtime, updatetime, id, campid, name, isprototype) VALUES
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000400000000', null, 'PBS Aufbaukurs Pfadistufe', true)
+;
+INSERT INTO public.checklist_item (createtime, updatetime, id, checklistid, parentid, position, text) VALUES
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000400010000', '000400000000', null, 1, 'Der Kurs ermöglicht den TN die Auseinandersetzung mit den Pfadigrundlagen.'),
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000400010001', '000400000000', '000400010000', 1, 'Vertiefung über die Bedürfnisse der Kinder und Judenlichen der Pfadistufe'),
```
Comment (BacLuc) [inline api/migrations/schema/checklists/pbs-checklist.sql]: google gemini empfiehlt:
```suggestion
    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000400010001', '000400000000', '000400010000', 1, 'Vertiefung über die Bedürfnisse der Kinder und Jugendliche der Pfadistufe'),
```

File: api/migrations/schema/checklists/pbs-checklist.sql [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000400010002', '000400000000', '000400010000', 2, 'Pfadigrundlagen als Hilfsmittel zum Sicherstellen der Ausgewogenheit des Programms'),
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000400010003', '000400000000', '000400010000', 3, '…Einbindung der sieben Methoden und fünf Pfadfinderbeziehungen in das Programm. als Mittel zur Erreichung der Ziele der Wölflingsstufe'),
```
Comment (BacLuc) [inline api/migrations/schema/checklists/pbs-checklist.sql]:
```suggestion
    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000400010003', '000400000000', '000400010000', 3, '…Einbindung der sieben Methoden und fünf Pfadfinderbeziehungen in das Programm als Mittel zur Erreichung der Ziele der Wolfsstufe'),
```

File: api/migrations/schema/checklists/pbs-checklist.sql [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000200030004', '000200000000', '000200030000', 4, 'Sicherheitskonzepte für sicherheitsrelevante Aktivitäten (inkl. herausfordernde Fähnliaktivitäten) planen und umsetzen'),
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000200030005', '000200000000', '000200030000', 5, 'Angebote und Anlaufstellen des Kantonalverbands/der Region inkl. Krisenkonzept'),
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000200030006', '000200000000', '000200030000', 6, 'eigene Leiterpersönlichkeit und Rolle im Team'),
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000200030007', '000200000000', '000200030000', 7, 'Regeln für konstruktive Gespräche im Leitungsteam'),
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000200030008', '000200000000', '000200030000', 8, 'Möglichkeiten der Aus- und Weiterbildung'),
+    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000200030009', '000200000000', '000200030000', 9, 'Sexuelle Ausbeutung und Grenzverletzungenund, mögliche heikle Situationen in Aktivitäten und verbeugende Massnahmen'),
```
Comment (BacLuc) [inline api/migrations/schema/checklists/pbs-checklist.sql]:
```suggestion
    ('2024-09-28 10:00:00', '2024-09-28 10:00:00', '000200030009', '000200000000', '000200030000', 9, 'Sexuelle Ausbeutung, Grenzverletzungen und mögliche heikle Situationen in Aktivitäten und verbeugende Massnahmen'),
```

**Review by BacLuc — COMMENTED** [review]
sieht gut aus, hab gemini hat noch kleine fehler gefunden

**Review by usu — COMMENTED** [review]
- It would be the first time we introduce sequential IDs. Any potential issue with that?
- Should we also update `migrations/prod/data.sql`? Not sure what the latest decision/discussion was on this for similar migrations.

**Comment by pmattmann** [comment]
> * It would be the first time we introduce sequential IDs. Any potential issue with that?

I don't think so.

> * Should we also update `migrations/prod/data.sql`? Not sure what the latest decision/discussion was on this for similar migrations.

It is actually user data. Therefore not 'seed'. Not really migration either.
But as long as we do not have a backend for external parties (PBS, Jubla), they can at least provide `sql`-files.

**Review by usu — APPROVED** [review]

**Review by BacLuc — APPROVED** [review]

**Review by simfeld — APPROVED** [review]

**Review by manuelmeister — APPROVED** [review]

**Pull request merged**